doc/user_guide/configuration/all-options.rst

```diff
@@ -1285,7 +1285,7 @@ Standard Checkers
 
 --spelling-dict
 """""""""""""""
-*Spelling dictionary name. Available dictionaries: en (aspell), en_AU (aspell), en_CA (aspell), en_GB (aspell), en_US (aspell).*
+*Spelling dictionary name. No available dictionaries : You need to install both the python package and the system dependency for enchant to work..*
```

Comment:
Can we remove this change?

Reply:
I created another commit for this: https://github.com/pylint-dev/pylint/pull/8899/commits/37786d8760cd14fa682aca4f10c57f12ee37378c 
```
 [doc] Upgrade the generated output for python 3.11 without enchant

The default needs to be that you need to install the enchant dict in the doc
```
I conscientiously restored the original value for months to keep it before that, but I think no dict installed make more sense as the default, you're more likely to need the installation info than the info that some dict can be installed.

Reply:
But I can open a new MR if this is not consensual